packages/leancode_contracts_generator/lib/src/contracts_generator.dart

```diff
@@ -91,7 +102,7 @@ class ContractsGenerator {
     // TODO: knownGroups?
     // TODO: projectName?
```

Comment:
As an "external" person, I've no idea what these comments mean. Maybe create GH Issues out of them and add a little more context?